src/scss/includes/routeLabel.scss

```diff
@@ -117,20 +121,16 @@
   }
 }
 
-.routeLabel--publicTransport {
-  .routeLabel-duration {
-    padding-top: 2px;
-  }
+.directions-stepByStep .routeLabel {
+  display: none;
+}
 
-  &.active .routeLabel-duration {
-    padding-top: 10px;
-  }
+@media (max-width: 640px) {
+  .routeLabel {
```

Comment:
I think we don't want to do that for transport modes other than public transport.